verifier/src/composer.rs

```diff
@@ -79,11 +79,11 @@ impl<E: FieldElement> DeepComposer<E> {
                 let value = E::from(value);
                 // compute the numerator of T'_i(x) as (T_i(x) - T_i(z)), multiply it by a
                 // composition coefficient, and add the result to the numerator aggregator
-                t1_num += (value - ood_main_trace_states[0][i]) * self.cc.trace[i].0;
+                t1_num += (value - ood_main_trace_states[0][i]) * self.cc.trace[i];
```

Comment:
I think we should also update comments on lines 49 - 51 above.

Reply:
Thanks